shuffle/cli/src/test.rs

```diff
@@ -127,3 +157,43 @@ fn run_deno_test(project_path: &Path, config: &NodeConfig) -> Result<()> {
         .wait()?;
     Ok(())
 }
+
+fn run_move_unit_tests(project_path: &Path) -> Result<()> {
+    let main_path = project_path.join("main");
+    Command::new("move")
+        .args(&["package", "test"])
```

Comment:
We don't want to invoke a separate process to run `move package test`. Let's call the rust code directly.